src/test_utils/database.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * @adonisjs/lucid
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { ApplicationService } from '@adonisjs/core/types'
+
+/**
+ * Database test utils are meant to be used during testing to
+ * perform common tasks like running migrations, seeds, etc.
+ */
+export class DatabaseTestUtils {
+  constructor(
+    protected app: ApplicationService,
```

Comment:
DatabaseTestUtils now accept `app` as parameter. That way we can lazily resolve `ace` only when needed